Fabric.Authorization.Persistence.SqlServer/EntityModels/Group.cs

```diff
@@ -9,11 +9,11 @@ public class Group : ITrackable, ISoftDelete
         public Group()
         {
             GroupRoles = new List<GroupRole>();
-            Users = new List<User>();
+            UserGroups = new List<UserGroup>();
```

Comment:
Should we be consistent with the naming convention when referencing join entities?

Reply:
Where is the inconsistency?

Reply:
We use GroupRoles in one case, and UserGroups in another case. It might be easier if we are consistent with the order if possible, i.e. parent then children, e.g. GroupRoles and GroupUsers...